### DISCUSSION TITLE
Micro-boluses stopped showing up since 2022-06-03

### DISCUSSION THREAD

**Discussion by mcurly** [NightscoutFoundation/xDrip#2163] [Q&A]
Hello everyone!

Is anybody else experiencing this sort of problem?

Since last Friday, 2022-06-03, xdrip stopped displaying several items among them are the micro boluses, but that's not all.

It has stopped showing up any treatments from fellow app, AndroidAPS.

Can anyone shed some light into this?

Thank you.

Before:

![Screenshot_20220605-073224_xDrip+](https://user-images.githubusercontent.com/8215020/172039065-6aa57ec4-2319-4a56-be8b-ef8aa440e834.png)

Now:

![Screenshot_20220605-075008_xDrip+](https://user-images.githubusercontent.com/8215020/172039099-31301898-5789-4039-83ca-1eb06d7b7534.png)

**Comment by Navid200** [comment]
@psonnera Any ideas?

**Comment by psonnera** [comment]
@mcurly how is xDrip+.configured with NSClient inter-app? I see you have basal meaning it's working. Do you have these enabled? https://xdrip.readthedocs.io/en/latest/use/interapp/#local-broadcast

**Reply by psonnera** [reply]
Accept glucose shouldn't be necessary.

**Reply by psonnera** [reply]
No, thinking about it if you have basal that means it's ok on xDrip+ side. Do treatments show if you enter them manually in xDrip+? Might be on NSClient side, but same there, if you receive basal rates that means broadcast to xDrip+ is enabled...

**Comment by mcurly** [comment]
Hello, thank you @psonnera and @Navid200 for your interest.

You're right, it doesn't show on nightscout rendering either.

What do you say? Should this be addressed somewhere else instead?

Thank you very much for your insights!

**Reply by psonnera** [reply]
I'd then start reviewing AAPS settings. I guess you're using a Libre sensor, AAPS should upload to Nightscout.

**Reply by mcurly** [reply]
Hello @psonnera !!!

Update:

Not quite in its settings but actually in the "NSCI" "dialog" window (from AAPS, obviously)...

Check out the attached images.

After unpausing NSCI all things took their course.

I did notice a warning (in AAPS) that I dismissed and that it showed by the time this oddity began to happen. Normally I don't disregard those warnings for NSCI malfunctioning but that time I did by distraction.

Sorry for having kept your precious time and efforts.

Cheers!!

![Screenshot_20220606_111036](https://user-images.githubusercontent.com/8215020/172142539-b1d3d0ee-8c92-4d06-bbf0-b8d934669790.png)


![Screenshot_20220606_110304](https://user-images.githubusercontent.com/8215020/172142670-0db911b6-a7fb-4936-a05d-6f8a014ae5c6.png)
![Screenshot_20220606_110429](https://user-images.githubusercontent.com/8215020/172142675-4bc0dfd6-2fa9-4034-ba5f-c455427f02cf.png)
![Screenshot_20220606_110554](https://user-images.githubusercontent.com/8215020/172142678-ffc91c67-aa47-4fcb-9827-6b51d1d07d8a.png)

**Reply by psonnera** [reply]
Glad you got it sorted!

**Reply by mcurly** [reply]
You helped a lot!

Thank you!
